pkg/version/version.go

```diff
@@ -130,3 +132,33 @@ func findTriggersVersion(deployments []v1.Deployment) string {
 	}
 	return version
 }
+
+// GetDashboardVersion Get dashboard version.
+func GetDashboardVersion(c *cli.Clients) (string, error) {
+	deploymentsList, err := getDeployments(c, dashboardControllerSelector, oldDashboardControllerSelector)
+
+	if err != nil {
+		return "", err
+	}
+
+	version := findDashboardVersion(deploymentsList.Items)
+	if version == "" {
+		return "", fmt.Errorf("Error getting the tekton dashboard deployment version. Version is unknown")
```

Comment:
Nit: Lowercase this error message

```suggestion
		return "", fmt.Errorf("error getting the tekton dashboard deployment version: version is unknown")
```

Reply:
Thanks for the review! Actually I copied the error handling from above (pipeline and trigger). 😂
Should they also be fixed?

Reply:
Yes, not your fault. There have definitely been misses about enforcing go error practices in reviews. You are more than welcome to update the other errors, but not a blocker to update them.

Reply:
I get it. Thank you very much! 😄